src/frontend/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "tsc --noEmit && webpack",
     "build-dev": "yarn copy-iframe-resizer; yarn build",
+    "clean": "find ../backend/marsha/static/js/ -type f ! -iname 'iframeResizer.min.js' -delete",
```

Comment:
Why not just move buid files into a `build` folder and simply remove content of this folder? From my point of view it will avoid to have to maintain this command if you add other js files into the `/static/js` folder.

Reply:
That's reminding me of your PR on richie, @jbpenrath  : https://github.com/openfun/richie/pull/1332 seems related to me

Reply:
Yep and I'm still hesitated to use a `clean` command like Manu did here ^^

Reply:
:+1: can I do it in an other PR ?